analytics/client.go

```diff
@@ -149,6 +167,69 @@ func newMixpanelClient(config Config) (mixpanel.Mixpanel, error) {
 	return mixpanel.New(config.MixpanelToken, mixpanelURL), nil
 }
 
+func newAmplitudeClient(config Config) (amplitude.Client, amplitude.EventOptions, error) {
+	if !config.IsAmplitudeEnabled || config.AmplitudeConfig == (AmplitudeConfig{}) {
```

Comment:
Why provide two different ways of disabling the Amplitude client? If the configuration has explicitly specified that Amplitude is enabled, shouldn't the absence of a configuration be an error? **Blocking.**